app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from __future__ import annotations
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    vaccine_error_counter = 0
+    mask_error_counter = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            vaccine_error_counter += 1
```

Comment:
You don't need the vaccine_error_counter. You can return here the error message straight away.